static/reference/next/client/README.md

```diff
@@ -1 +1,28 @@
-> Docs for Pulsar Java Client Configuration.
+## Client Configurations
+
+If you create a Java client, producer, consumer, or reader, you can use the `loadConf` configurations.
+
+- [Client configurations](/next/client/client-configuration-client)
+- [Producer configurations](/next/client/client-configuration-producer)
+- [Consumer configurations](/next/client/client-configuration-consumer)
+- [Reader configurations](/next/client/client-configuration-reader)
+
+### Set client memory allocator configs using Java system properties
+
+You can set the client memory allocator configurations through Java properties.<br />
+
+| Property | Type | <div style="width: 110pt"> Description </div> | Default | Available values |
```

Comment:
`110pt` is a compromised value when I was trying to expand the width of this column to include more characters in one row. Higher values will expand the entire table, and users have to scroll horizontally to see all the information. If there are any other better solutions, please let me know.